### ISSUE TITLE
why does skywalking not support latest version of opentracing

### ISSUE THREAD

**Issue by 844067874** [apache/skywalking#7371]
Please answer these questions before submitting your issue.

- Why do you submit this issue?
- [x] Question or discussion
- [ ] Bug
- [ ] Requirement
- [ ] Feature or performance improvement

___
### Question
- What do you want to know?
why does skywalking not support the latest version of opentracing,the latest version is 0.33.0, it just support 0.30.0 in `apm-toolkit-opentracing`
___
### Bug
- Which version of SkyWalking, OS, and JRE?

- Which company or project?

- What happened?
If possible, provide a way to reproduce the error. e.g. demo application, component version.

___
### Requirement or improvement
- Please describe your requirements or improvement suggestions.

**Comment by wu-sheng** [comment]
First and most importantly, no PMC care about that project anymore. Because no one actually is using it.
And OpenTracing has a bad practice about forwarding compatibility. All APIs are breaking, again and again, we can't see any benefit from it.

**Comment by wu-sheng** [comment]
If you want, you could try, but for real, its latest APIs are very hard to follow from SkyWalking perspective. We tried 2 years ago, but fail.
